docs/reference/commandline/swarm_leave.md

```diff
@@ -31,7 +31,7 @@ dkp8vy1dq1kxleu9g4u78tlag    worker1   Ready   Active        Reachable
 dvfxp4zseq4s0rih1selh0d20 *  manager1  Ready   Active        Leader
 ```
 
-On a worker node:
+On worker2 node:
```

Comment:
ah, yes, the original text basically says on _a_ worker node (which happens to be `worker2` in this case.

I'm ok with it but perhaps;

```Markdown
On a worker node (`worker2` in this example):
```

ping @sfsmithcha any suggestions?